integration-tests/spring-boot-properties/README.md

```diff
@@ -0,0 +1,12 @@
+### Current differences with Spring Boot properties.
+
+1. Default values assigned with `=` operator are not supported.
+2. Getters and setters are needed for all fields, including nested classes.
+3. `@Bean` needs to be removed from a method which is annotated with `@ConfigurationProperties`.
```

Comment:
It shouldn't be hard to make this work - basically by just ignoring `@Bean`. We should follow up this PR with that